docs/getting-started.md

```diff
@@ -143,7 +143,7 @@ Import the `protect` function and initialize a client with your defined schema,
 
 ```ts
 import { protect } from '@cipherstash/protect'
-import { users } from './schema'
+import { users, orders } from './schema'
```

Comment:
This was fixed in the unmerged [PR](https://github.com/cipherstash/protectjs/pull/110) but a different way.
Is it clearer to include `orders`? If so I can close #110.

Reply:
Yeah, I reckon we leave `orders`. Have closed #110.